skype/skype-ps/skype/skype.md

```diff
@@ -2710,6 +2722,9 @@ The following cmdlet references are for Skype for Business and Microsoft Teams.
 ### [Set-CsTeamsMeetingPolicy](Set-CsTeamsMeetingPolicy.md)
 {{Manually Enter Set-CsTeamsMeetingPolicy Description Here}}
```

Comment:
You need to copy the sentence from the SYNOPSIS to replace the placeholder text

Reply:
Done, copied short cmdlet description to skype.md